lib/hawkular/operations/operations_api.rb

```diff
@@ -53,14 +53,29 @@ def to_msg_hash
     # @example
     #   Hawkular::Operations::OperationsClient.new(credentials: {username: 'jdoe', password: 'password'})
     def initialize(args)
-      args[:host] ||= 'localhost:8080'
+      ep = args[:entrypoint]
```

Comment:
On 27 Jun 2016, at 12:08, abonas wrote:

>> @@ -53,14 +53,29 @@ def to_msg_hash
>>      # @example
>>      #   Hawkular::Operations::OperationsClient.new(credentials: 
>> {username: 'jdoe', password: 'password'})
>>      def initialize(args)
>> -      args[:host] ||= 'localhost:8080'
>> +      ep = args[:entrypoint]
>
> btw I'm confused where is entrypoint coming from? I read lines 44-51 
> and don't see it there

I'll add it to the javadoc.

It is passed from here 
https://github.com/pilhuhn/hawkular-client-ruby/blob/f27efffd42de858a9fc1238dcc693a62abd2b7e2/lib/hawkular/hawkular_client.rb#L61
when one does

```
client = Hawkular::Client.new()
ops_client = client.operations()
```

As this `init_operations_client()` never passed `args[:host]` the 
default of `localhost:8080` kicked in :-(